### ISSUE TITLE
SqlServer.Management.Smo.FailedOperationException when column name contains a quote...

### ISSUE THREAD

**Issue by natiki** [thiagotts/parilis#2]
Hi,

Got the following exception when preparing the information about the database I was interested in:

![image](https://cloud.githubusercontent.com/assets/4738140/8790160/c78edd14-2f8d-11e5-8e37-428f1d6e36f3.png)

The SQL that was being executed was: 

```
"SELECT IS_NULLABLE, columnproperty(object_id('dbo._RCCL'),'First Guest's Last Name','IsIdentity'), DATA_TYPE, CHARACTER_MAXIMUM_LENGTH\r\n                                        FROM INFORMATION_SCHEMA.COLUMNS\r\n                                        WHERE TABLE_NAME = '_RCCL'\r\n                                        AND TABLE_SCHEMA = 'dbo'\r\n                                        AND COLUMN_NAME = 'First Guest's Last Name'"
```

As the table in question has quotes in the field names I suspect that is what is causing this to blow up.

![image](https://cloud.githubusercontent.com/assets/4738140/8790170/f7197c88-2f8d-11e5-9b4f-cb07c62483d8.png)

**Comment by thiagotts** [comment]
Bug is confirmed. Fix is on the way!

**Comment by thiagotts** [comment]
Bug was fixed in [release 0.3](https://github.com/thiagotts/parilis/releases/tag/0.3).

**Comment by natiki** [comment]
Confirm that this now works on my end with 0.3.
